Core/nextspace-core.spec

```diff
@@ -45,6 +45,7 @@ Requires:	make
 Requires:	git
 Requires:	patch
 Provides:	gnustep-make
+BuildRequires:	autoconf
```

Comment:
What package part of project needs this?

Reply:
What the problem with existing `configure`. It works as is.

Reply:
As mentioned below:

The problem with the `AC_PATH_PROG` macro is that it behaves differently if you are building as root as directories are searched in a different order. In my case it located `bash` in `/usr/sbin` and generated an RPM dependency to `/usr/sbin/bash`.